xcode/Subconscious/Shared/Components/Common/Audience/MenuButtonView.swift

```diff
@@ -19,19 +19,21 @@ struct MenuButtonView<Icon: View>: View {
             Text(label)
                 .bold()
                 .font(.caption)
+            Spacer()
             Image(systemName: "chevron.down")
                 .font(.system(size: 12))
+                .foregroundColor(Color.secondary)
         }
+        .foregroundColor(Color.accentColor)
         .lineLimit(1)
-        .frame(height: AppTheme.unit * 6)
-        .foregroundColor(Color.primaryButtonText)
+        .frame(height: 30)
```

Comment:
> Also worth mentioning: increasing the system font size will make the font bigger here but will not increase the size of the frame, leading to a cramped appearance. Non-blocking but just an FYI

@bfollington interesting catch. Unfortunately this item must have a hard-coded width for the transition between Everyone/Local to work right. Otherwise the text reflows and we get a janky animation. Doing better than this will probably require defining some dynamic unit that is responsive to system font size.

Filed an issue to pursue that in a followup PR https://github.com/subconsciousnetwork/subconscious/issues/436

Reply:
> Does this correspond to a standard size? If not, should we add it to AppTheme as a constant? Both 44 (because Apple HIG), 32 and 30 all seem like good candidates for constants.

I did not specify this as a constant, because I want to discourage casual use of anything smaller than 44 (minimum touch target). This control is a bespoke exception, because it is wider than 44, making it easy to hit. It's based roughly off of similar selector controls seen in Notes app and elsewhere.

Reply:
All good 👍